src/features/Table/index.tsx

```diff
@@ -3,20 +3,44 @@
 // import SearchBar from '../SearchBar';
 import Row from './Row';
 
-const Table = (): JSX.Element => {
+export interface TableEmployee {
```

Comment:
That's correct ,this is the interface for table employee. Since you were able to find out by the name I don't see an issue :). 

But feel free to suggest a name , and I will adjust it

Reply:
This is not the interface for the table employee. It represent the row of the table. Once you export this interface outside of the file, you lost all the meaning you had inside the file. As an example, [this line from DeleteForm](https://github.com/Chrysalis-Papilio/Soen490-Papilio-Website/blob/6fcc0a5f7dd324db793924b686fedebd7b4b36f0/src/pages/Dashboard/Employees/DeleteForm/index.tsx#L6) will not be interpreted as you want. 
```
export declare interface DeleteFormInterface {
  employees: TableEmployee[]
  onSubmit: (employeeIds: number[]) => void
}
```
Now, when looking at the DeleteFormInterface, you will think that you have a table per employee. This is a required change because it has taken me more than 5 minutes to understand the meaning of this variables. In a month, when you read back this code, you will have trouble understanding what this variable mean and will have to investigate through all the code to understand the meaning of TableEmployee